components/automate-gateway/api/iam/v2beta/rules.proto

```diff
@@ -38,4 +38,12 @@ service Rules {
     option (chef.automate.api.iam.policy).resource = "iam:rules";
     option (chef.automate.api.iam.policy).action = "iam:rules:list";
   };
+
+  rpc DeleteRule (DeleteRuleReq) returns (DeleteRuleResp) {
+    option (google.api.http).delete= "/iam/v2beta/rules/{id}";
```

Comment:
[super nit] `s/delete=/delete =` Looks like we messed this up in a few other places too.